src/NServiceBus.Persistence.AzureStorage/SagaPersisters/SagaIdGenerator.cs

```diff
@@ -0,0 +1,44 @@
+﻿namespace NServiceBus.Persistence.AzureStorage
+{
+    using System;
+    using Sagas;
+    using System.Security.Cryptography;
+    using System.Text;
+    using Newtonsoft.Json;
+
+    class SagaIdGenerator : ISagaIdGenerator
+    {
+        public Guid Generate(SagaIdGeneratorContext context)
+        {
+            if (context.CorrelationProperty == SagaCorrelationProperty.None)
+            {
+                // TODO: verify!
```

Comment:
This has been the case in the past. Is it still the case or no longer?

Reply:
Custom saga finder means multiple properties. We can't do scan on multiple properties.